Draft publishing workflow documentation (#605)
directus/docs: content/guides/01.data-model/1.collections.md

Content versioning is used by :product-link{product="editor"} and allows teams to create and manage different versions of their content. There are several reasons to use content versioning, including drafting content without publishing it, and more ways to collaborate with others.

> This feature can be enabled for specific collections, and will be available for all items. Once enabled, item versions can be created, and later have some or all fields published to the main version.

Reviewer: Should we stop referencing **main version** here as well? Given we'vee replaced that with published throughout the app.

Suggestion:
> This feature can be enabled for specific collections. Once enabled, new items open automatically in a draft version, where edits can be made and saved without affecting the published content. When ready, the draft can be published — promoting it to the published version.

Live preview is used by :product-link{product="editor"} and allows for your application to be shown in a pane next to your content, which can be used for previewing content before publishing.

> You can use item field values to construct the URL used by the live preview, including unique identifiers and content version, allowing for previewing content versions before publishing them to the main version.

Reviewer: ```suggestion
Use item field values to construct the live preview URL, including unique identifiers and the content version, so you can preview draft content before publishing it.
```
